client/javascripts/actions/authActionCreators.js

```diff
@@ -0,0 +1,4 @@
+import { createAction } from 'redux-actions';
```

Comment:
__[eslint]__
* Error - Resolve error
* Error - Unable to resolve path to module 'redux-actions'. (import/no-unresolved)
* Error - Missing file extension for "redux-actions" (import/extensions)

<!-- Comment by [SideCI](https://sideci.com) -->